packages/cli-kit/src/public/node/ui.tsx

```diff
@@ -44,6 +47,7 @@ export async function renderConcurrent({
   )
 }
 
+export type RenderAlertCutomSection = CustomSection
```

Comment:
Could we rename this to `AlertCustomSection` to keep it consistent with the other exports? The type is more tied to the component than the render method itself.